packages/search-ui/src/Results/components/Result/types.ts

```diff
@@ -14,6 +14,8 @@ interface Props extends Pick<ResultsProps, 'appearance' | 'imageAspectRatio' | '
   onClick?: UseClickTrackingParams['onClick'];
   /** Display image or not */
   showImage?: boolean;
+  /** Display variant images or not */
+  showVariantImage?: boolean;
```

Comment:
Since it might be relevant to the price change, probably name it `showVariant` makes more sense?

```suggestion
  showVariant?: boolean;
```

Reply:
I don't think so, `showVariantImage` is solely being used to show images, price change is just a result of hovering on each image, plus `showVariant` might not sound specific enough when first read (variant could be anything).

Reply:
But we can only have the price change if `showVariantImage` is enabled. Is that right? My point here is that `showVariantImage` doesn't reflect the fact that the price could change.